src/context/card-context.js

```diff
@@ -11,16 +12,23 @@ export const CardContext = React.createContext({
 });
 
 const CardContextProvider = props => {
-  const [cards, setCards] = useState([
-    {headerData: 'Caption 1', bodyData: 'text 1', id: 1},
-    {headerData: 'Caption 2', bodyData: 'text 2', id: 2},
-    {headerData: 'Caption 3', bodyData: 'text 3', id: 3},
-    {headerData: 'Caption 4', bodyData: 'text 4', id: 4},
-    {headerData: 'Caption 5', bodyData: 'text 5', id: 5},
-    {headerData: 'Caption 6', bodyData: 'text 6', id: 6},
-    {headerData: 'Caption 7', bodyData: 'text 7', id: 7},
-    {headerData: 'Caption 8', bodyData: 'text 8', id: 8}
-  ]);
+  const [cards, setCards] = useState([]);
+
+  useEffect(() => {
+    axios.get('https://raw.githubusercontent.com/BrunnerLivio/PokemonDataGraber/master/output.json')
+         .then(response => {
+           const data = response.data.slice(0, 15);
+           const updatedData = data.map(item => {
+             return {
+               headerData: item.Name,
+               bodyData: item.About,
+               id: item.Number
+             }
+           });
+           setCards([...updatedData]);
```

Comment:
Actually this could be simplified. There is no need in a constant/variable if you're using smth just once. Also you can use arrow function to create even shorter return statement:
```
setCards(
  response.data.slice(0, 15)
    .map(item => ({
      headerData: item.Name,
      bodyData: item.About,
      id: item.Number,
})));
```